### PR TITLE
pull request for "adding X_FORWARDED_PREFIX for wsgi compatible apps #216"

### PR DESCRIPTION
**Pull request by gbrault**
This enable to have flask/werkzeug apps seamless integration: see issue #216

### CODE REVIEW

File: jupyter_server_proxy/handlers.py [hunk 1]
```diff
@@ -172,6 +172,8 @@ def _build_proxy_request(self, host, port, proxied_path, body):
             context_path = self._get_context_path(port)
             headers['X-Forwarded-Context'] = context_path
             headers['X-ProxyContextPath'] = context_path
+            # to be compatible with flask/werkzeug wsgi applications
+            headers['X_FORWARDED_PREFIX'] = context_path
```
Comment (yuvipanda) [inline jupyter_server_proxy/handlers.py:176]: Should this be 'X-Forwarded-Prefix'? That is the more common HTTP Header naming convention. https://werkzeug.palletsprojects.com/en/1.0.x/middleware/proxy_fix/ seems to say that, but not sure if I'm looking in the right part of the documentation.

**Review by yuvipanda — CHANGES_REQUESTED** [review]
Thank you for opening this PR!

**Review by yuvipanda — CHANGES_REQUESTED** [review]
Thank you for opening this PR!

File: jupyter_server_proxy/handlers.py (hunk 1 above)
Comment (gbrault) [reply jupyter_server_proxy/handlers.py:176]: you are right from the reference, but W3C says [Headers are case-insensitive](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html). do you want me to fix it?
Please let me now, I'll do another PR.
NB: I have a working case under test..

Comment (yuvipanda) [reply jupyter_server_proxy/handlers.py:176]: Ah, I was more worried about the '_' instead of '-', rather than the case.

Comment (gbrault) [reply jupyter_server_proxy/handlers.py:176]: Ok, my mistake, I verify!

Comment (gbrault) [reply jupyter_server_proxy/handlers.py]: Funy enough, it's what I have (and working well) on my test platform: I make the change and verify it works.

Comment (gbrault) [reply jupyter_server_proxy/handlers.py]: Just tested the change and it works as well! 

Do I make a new pull request?

File: jupyter_server_proxy/handlers.py [hunk 2]
```diff
@@ -172,6 +172,9 @@ def _build_proxy_request(self, host, port, proxied_path, body):
             context_path = self._get_context_path(port)
             headers['X-Forwarded-Context'] = context_path
             headers['X-ProxyContextPath'] = context_path
+            # to be compatible with flask/werkzeug wsgi applications
+            # headers['X_FORWARDED_PREFIX'] = context_path
```
Comment (yuvipanda) [inline jupyter_server_proxy/handlers.py:176]:
```suggestion
```

**Review by yuvipanda — COMMENTED** [review]
Awesome! Let's just remove the commented out line, and I can merge this.

**Comment by yuvipanda** [comment]
Thanks a lot for your patch, @gbrault! I hope this is useful for you, and you contribute more patches in the future! <3

**Comment by gbrault** [comment]
Thanks guys!

**Pull request merged**